### PR TITLE
update several sql statement docs

### PR DESCRIPTION
**Pull request by bb7133**
### What is changed, added or deleted? (Required)
update several sql statement docs:
 - sql-statements/sql-statement-drop-sequence.md
 - sql-statements/sql-statement-show-character-set.md
 - sql-statements/sql-statement-show-collation.md

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: sql-statements/sql-statement-drop-sequence.md [hunk 1]
```diff
@@ -15,10 +15,6 @@ aliases: ['/docs-cn/dev/reference/sql/statements/drop-sequence/']
 
 ![DropSequenceStmt](/media/sqlgram/DropSequenceStmt.png)
 
-**OptTemporary:**
-
-![OptTemporary](/media/sqlgram/OptTemporary.png)
-
 **IfExists:**
 
 ![IfExists](/media/sqlgram/IfExists.png)
```
Comment (djshow832) [inline sql-statements/sql-statement-drop-sequence.md:20]: This diagram is wrong. The empty branch is missing.

Comment (bb7133) [reply sql-statements/sql-statement-drop-sequence.md:20]: Done

**Comment by bb7133** [comment]
PTAL @djshow832

File: sql-statements/sql-statement-show-character-set.md [hunk 2]
```diff
@@ -46,7 +42,7 @@ SHOW CHARACTER SET;
 
 ## MySQL 兼容性
 
-`SHOW CHARACTER SET` 语句与 MySQL 完全兼容。如有任何兼容性差异，请在 GitHub 上提交 [issue](/report-issue.md)。
+`SHOW CHARACTER SET` 语句功能与 MySQL 完全兼容。然而请注意，TiDB 中字符集的默认排序规则与 MySQL 有所不同，详情请见[与 MySQL 兼容性对比](/mysql-compatibility.md)。对于其他问题导致的兼容性差异，请在 GitHub 上提交 [issue](/report-issue.md)。
```
Comment (zimulala) [inline sql-statements/sql-statement-show-character-set.md:45]: 是不是可以直接连接到 "mysql-compatibility.md" 文件的具体段落？

Comment (bb7133) [reply sql-statements/sql-statement-show-character-set.md:45]: 我觉得还好吧，这里已经比较简略了

File: sql-statements/sql-statement-show-collation.md [hunk 3]
```diff
@@ -9,256 +9,61 @@ aliases: ['/docs-cn/dev/reference/sql/statements/show-collation/']
 
 `SHOW COLLATION` 语句用于提供一个静态的排序规则列表，确保与 MySQL 客户端库的兼容性。
 
-> **注意:** TiDB 目前仅支持二进制排序规则。
+> **注意:** `SHOW COLLATION` 所展示的排序规则列表与是否 TiDB 集群是否开启新排序规则框架有关，详情请见 [TiDB 字符集和排序规则](/character-set-and-collation.md))
```
Comment (toutdesuite) [inline sql-statements/sql-statement-show-collation.md:12]:
```suggestion
> **注意：**
>
> `SHOW COLLATION` 所展示的排序规则列表与 TiDB 集群是否开启新排序规则框架有关，详情请见 [TiDB 字符集和排序规则](/character-set-and-collation.md))
```

File: sql-statements/sql-statement-show-collation.md [hunk 4]
```diff
@@ -9,256 +9,61 @@ aliases: ['/docs-cn/dev/reference/sql/statements/show-collation/']
...
-
-![ShowTargetFilterable](/media/sqlgram/ShowTargetFilterable.png)
+![ShowCollationStmt](/media/sqlgram/ShowCollationStmt.png)
 
 ## 示例
 
 {{< copyable "sql" >}}
 
 ```sql
+" 若未开启新排序规则框架，仅展示二进制排序规则
```
Comment (toutdesuite) [inline sql-statements/sql-statement-show-collation.md:25]:
```suggestion
```

Comment (toutdesuite) [reply sql-statements/sql-statement-show-collation.md:25]: It is not recommended to add comments into code blocks. Please move the comments to L22~

File: sql-statements/sql-statement-show-collation.md [hunk 5]
```diff
@@ -9,256 +9,61 @@ aliases: ['/docs-cn/dev/reference/sql/statements/show-collation/']
...
+| latin1_bin  | latin1  |   47 | Yes     | Yes      |       1 |
+| binary      | binary  |   63 | Yes     | Yes      |       1 |
+| ascii_bin   | ascii   |   65 | Yes     | Yes      |       1 |
+| utf8_bin    | utf8    |   83 | Yes     | Yes      |       1 |
++-------------+---------+------+---------+----------+---------+
+5 rows in set (0.02 sec)
+```
+
+```sql
+" 若开启了新排序规则框架，则在二进制排序规则之外，额外支持 utf8_general_ci 和 utf8mb4_general_ci 两种大小写和口音不敏感的排序规则
```
Comment (toutdesuite) [inline sql-statements/sql-statement-show-collation.md:44]: ditto

File: sql-statements/sql-statement-show-collation.md [hunk 6]
```diff
@@ -9,256 +9,61 @@ aliases: ['/docs-cn/dev/reference/sql/statements/show-collation/']
...
+| utf8mb4_bin        | utf8mb4 |   46 | Yes     | Yes      |       1 |
+| utf8mb4_general_ci | utf8mb4 |   45 |         | Yes      |       1 |
++--------------------+---------+------+---------+----------+---------+
+7 rows in set (0.00 sec)
 ```
 
 ## MySQL 兼容性
 
-TiDB 不支持二进制以外的排序规则。`SHOW COLLATION` 语句仅用于确保与 MySQL 的兼容性。
+`SHOW COLLATION` 语句功能与 MySQL 完全兼容。然而请注意，TiDB 中字符集的默认排序规则与 MySQL 有所不同，详情请见[与 MySQL 兼容性对比](/mysql-compatibility.md)。对于其他问题导致的兼容性差异，请在 GitHub 上提交 [issue](/report-issue.md)。
```
Comment (toutdesuite) [inline sql-statements/sql-statement-show-collation.md:66]:
```suggestion
`SHOW COLLATION` 语句功能与 MySQL 完全兼容。注意，TiDB 中字符集的默认排序规则与 MySQL 有所不同，具体可参考[与 MySQL 兼容性对比](/mysql-compatibility.md)。对于其他问题导致的兼容性差异，请在 GitHub 上提交 [issue](/report-issue.md)。
```

**Comment by bb7133** [comment]
Comment addressed, PTAL @toutdesuite @djshow832 @zimulala

File: sql-statements/sql-statement-show-collation.md [hunk 7]
```diff
@@ -9,256 +9,67 @@ aliases: ['/docs-cn/dev/reference/sql/statements/show-collation/']
...
+## 示例
 
-![ShowTargetFilterable](/media/sqlgram/ShowTargetFilterable.png)
+若未开启新排序规则框架，仅展示二进制排序规则：
 
-## 示例
+{{< copyable "sql" >}}
+
+```sql
+
```
Comment (yikeke) [inline sql-statements/sql-statement-show-collation.md:29]:
```suggestion
```

File: sql-statements/sql-statement-show-collation.md [hunk 8]
```diff
@@ -9,256 +9,67 @@ aliases: ['/docs-cn/dev/reference/sql/statements/show-collation/']
...
++-------------+---------+------+---------+----------+---------+
+5 rows in set (0.02 sec)
+```
+
+若开启了新排序规则框架，则在二进制排序规则之外，额外支持 `utf8_general_ci` 和 `utf8mb4_general_ci` 两种大小写和口音不敏感的排序规则：
 
 {{< copyable "sql" >}}
 
 ```sql
+
```
Comment (yikeke) [inline sql-statements/sql-statement-show-collation.md:51]:
```suggestion
```

File: sql-statements/sql-statement-show-character-set.md (hunk 2 above)
Comment (zimulala) [reply sql-statements/sql-statement-show-character-set.md]: 简略？我的意思是链接到了 "mysql-compatibility.md" 文件，但是字符集相关段落还要翻两页才能找到具体描述。

**Comment by yikeke** [comment]
Please resolve this comment: https://github.com/pingcap/docs-cn/pull/3294#discussion_r430265366 @bb7133

File: sql-statements/sql-statement-show-character-set.md (hunk 2 above)
Comment (bb7133) [reply sql-statements/sql-statement-show-character-set.md]: 理解了

Comment (bb7133) [reply sql-statements/sql-statement-show-character-set.md]: Done

**Comment by bb7133** [comment]
PTAL @yikeke @zimulala

**Pull request merged**